ubi_manifest/worker/tasks/depsolver/utils.py

```
@@ -6,11 +6,15 @@
 from typing import Dict, List, Tuple
 
 from pubtools.pulplib import Client, Criteria, Matcher, RpmDependency
-from rpm import labelCompare as label_compare  # pylint: disable=no-name-in-module
 from ubiconfig import UbiConfig
 
 from ubi_manifest.worker.tasks.depsolver.models import PackageToExclude
 
+try:
+    from rpm import labelCompare as label_compare  # pylint: disable=no-name-in-module
+except ImportError as ex:  # pragma: no cover
+    pass
```

Can you add some logging message here instead of silently passing?